java/java.source.base/src/org/netbeans/modules/java/source/save/Reformatter.java

```diff
@@ -2649,6 +2650,96 @@ public Boolean visitSwitch(SwitchTree node, Void p) {
             return true;
         }
 
+        public Boolean scanSwitchExpression(Tree node, Void p) {
```

Comment:
I assume this is basically a copy of visitSwitch - was there an attempt to not duplicate the code? Not sure if that'd be possible, but maybe something like:
handleSwitch(ExpressionTree expression, Iterable<? extends CaseTree> cases) {...}
that would be called from both places?

Reply:
Had moved the duplicate code as separate method handleSwitch()